project/backend/src/main/java/com/google/sps/data/RestaurantScorer.java

```diff
@@ -110,11 +110,11 @@ private static double metersToMiles(double numMeters) {
   /** Skews ratings so that ratings with a lower number of ratings are weighed less. */
   private static double calculateRatingScore(
       double avgRating, int numRatings, DescriptiveStatistics statistics) {
-    int numInitialRatings = (int) Math.round(statistics.getPercentile(10));
+    int numInitialRatings = (int) Math.round(statistics.getMean() / 5);
```

Comment:
Lets add the part of the PR description you wrote about why we're using mean/5 as comments here

Reply:
or I think what could work is describing the overall approach of this method in the method comment above where you can explain the rationale for this and other calculations below